src/renderer/components/Menu.js

```diff
@@ -44,7 +44,7 @@ class Controller {
     if (selectedChat && selectedChat.contacts.length) {
       var contact = selectedChat.contacts[0]
       var message = tx('dialogs.blockContact', contact.displayName)
-      dialogs.confirmation(message, yes => {
+      confirmation(message, yes => {
```

Comment:
another cool UI improvement step would to be to make a nicer confirmation dialog that follows our styling, too